code/search/interpolation_search/Interpolation.java

```diff
@@ -0,0 +1,45 @@
+package arrays;
```

Comment:
This line must be removed as an end user may not be using the package name and thus, will result in error.

Removing it makes the code perfect.